tests/python/cli/test_cli.py

```diff
@@ -302,3 +302,16 @@ def test_can_control_organization_context(self):
         all_task_ids = list(map(int, self.run_cli("ls").split()))
         assert personal_task_id in all_task_ids
         assert org_task_id in all_task_ids
+
+    def test_auto_annotate(self, fxt_new_task: Task):
+        annotations = fxt_new_task.get_annotations()
+        assert not annotations.shapes
+
+        self.run_cli(
+            "auto-annotate",
+            str(fxt_new_task.id),
+            f"--function={__package__}.example_function",
```

Comment:
Again, from a regular user perspective, I will recommend supporting at least several interfaces. I'm not sure that we have to force users to know what a Python module is. At the same time, many users know what is a directory. Do you think we can redesign the interface and use widely known concepts amount CLI users?

Reply:
I added an alternate option that takes a file path.